### DISCUSSION TITLE
Snoozz v2.3.4 beta

### DISCUSSION THREAD

**Discussion by rohanb10** [rohanb10/snoozz-tab-snoozing#32] [General]
#### 2.4.0
- Edit Snoozz times for sleeping tabs 🎉
- New Date and Time picker for custom snooze time selection
- Snooze selected tabs (and a new UI for selecting snooze targets)
- New options: "One Hour From Now" and "On Next Startup"
- Export and Smart Import tabs to use across different devices (No duplicates, the most recently updated tab will be saved to your device)

Lots of changes made to the popup, dashboard, and settings page so need to test a lot to ensure stability before publishing to the stores.

<hr /><em>This discussion was created from the release <a href='https://github.com/rohanb10/snoozz-tab-snoozing/releases/tag/v2.3.4-beta'>Snoozz v2.3.4 beta</a>.</em>

**Comment by hodanli** [comment]
how can i test the beta

**Reply by rohanb10** [reply]
Download the beta zip file directly from [here](https://github.com/rohanb10/snoozz-tab-snoozing/archive/refs/tags/v2.3.4-beta.zip) or the assets menu on [this page](https://github.com/rohanb10/snoozz-tab-snoozing/releases/tag/v2.3.4-beta) if you prefer 

Chrome
Skip the first step.
https://dev.to/ben/how-to-install-chrome-extensions-manually-from-github-1612

Firefox
https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/

**Reply by hodanli** [reply]
thnx. beta works without any issue for me on firefox.
